docs/features/requestaggregation.rst

```diff
@@ -108,6 +108,31 @@ In order to make an Aggregator you must implement this interface.
 
 With this feature you can pretty much do whatever you want because the HttpContext objects contain the results of all the aggregate requests. Please note if the HttpClient throws an exception when making a request to a Route in the aggregate then you will not get a HttpContext for it but you would for any that succeed. If it does throw an exception this will be logged.
 
+.. code-block:: csharp

```

Comment:
**There is no description of this sample!**
Developers need to see some instructions and best design approaches while implementing a custom aggregator class.